### ISSUE TITLE
Mobile friendly display

### ISSUE THREAD

**Issue by bloodybowlers** [bloodybowlers/UkeGeeks-ng#62]
There is currently no mobile/tablet version of the display.

Massive switch to bootstrap, or a simple hack to detect mobile and try to adapt the display ?

The problem with the current UkeGeeks display, imho, is that it is made to be easily printable.  
This enable printing rather easily, but it restrict the display a lot...

**Comment by Louis-coding-UGNG** [comment]
I'd vote for the detection method and build a layout for mobile only - this will duplicate some pages but is most probably less effort than a "all-in-one" solution.

On the other hand a mobile friendly version should IMHO also include minified css and js files to save bandwidth.

**Comment by bloodybowlers** [comment]
You are right, an all-in-one solution would be a pain, especially given the current state of the code...

We could try to make mobile version of login/song-list/song-edit page.  
But I think we need to make the song-edit page **read only**. I mean, who is using a mobile/tablet to add songs anyway ? It's a pain. You just need to be able to read it and play !

**Comment by Louis-coding-UGNG** [comment]
In another project I used https://github.com/serbanghita/Mobile-Detect for switching....

I am not so sure about the read-only. It should be default for mobiles, but not the only option. It does happen that you discover errors or have new ideas - and then you're quite happy to have an editing option in the back (even when it is a bit painful). When editing on mobile device the editor should be fullscreen (in a new tab?), not just the small overlay we have on desktop; the advanced editor seems too much....
Brainstorming:
- send mobile devices to the public "read-only" page with a mobile friendly layout
- leave the login option available
- mobile editing only in fullscreen mode (can we trigger a page refresh in another tab for previewing? Or can we skip such functionality for the time being?)
